packages/flutter/lib/src/material/search_anchor.dart

```diff
@@ -350,6 +351,9 @@ class SearchAnchor extends StatefulWidget {
   /// Defaults to the default value specified in [TextField].
   final TextInputType? keyboardType;
 
+  /// If false the widget is "disabled" so the SearchAnchor will ignore taps.
+  final bool enabled;
```

Comment:
I know these docs are the same as the other PR for SearchBar, but would you mind changing it there too?

```
/// Whether or not this widget is currently interactive.
///
/// When false, the widget will ignore taps and appear dimmed.
///
/// Defaults to true.
```